.travis.yml

```diff
@@ -28,17 +28,13 @@ addons:
       - pkg-config
       - protobuf-compiler
       - libprotobuf-dev
-      - liblz4-dev
-      - libzstd-dev
 
 env:
   global:
     - RUST_COMPILER_VERSION=stable
     - OSX_PACKAGES="libsodium rocksdb pkg-config protobuf"
     - ROCKSDB_STATIC=1
```

Comment:
You said on a call we link on Linux dynamically. But this env variable is shared between all jobs.

Reply:
Good catch, thanks. The configuration was invalid and this variable didn't work because no `ROCKSDB_LIB_DIR` was provided.